src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(callback, number) {
+      this.result = callback.call(this, this.result, number);
+
```

Comment:
```suggestion
      this.result = callback.call(this.result, number);

```

I think this value is redundant here